cdc/model/sink.go

```diff
@@ -1092,6 +1092,15 @@ func (d *DDLEvent) FromJobWithArgs(
 	case model.ActionDropView:
 		d.Query = fmt.Sprintf("DROP VIEW `%s`.`%s`",
 			d.TableInfo.TableName.Schema, d.TableInfo.TableName.Table)
+	case model.ActionRenameTable:
+		// Note: preTableInfo may not be accurate for rename table.
+		// after pr: https://github.com/pingcap/tidb/pull/43341,
+		// assume there is a table `test.t` and a ddl: `rename table t to test2.t;`, and its commit ts is `100`.
+		// if you get a ddl snapshot at ts `99`, table `t` is already in `test2`.
+		// so preTableInfo.TableName.Schema will also be `test2`.
```

Comment:
Does it impact rename multiple tables?

Reply:
No. Rename tables has it's unique way get these info.